apps/srt-live-transmit.cpp

```diff
@@ -534,6 +534,15 @@ int main(int argc, char** argv)
                         return 1;
                     }
                     break;
+                case UriParser::RTP:
```

Comment:
You can just as well add the case label under UDP as both resolving to the same handler. Copy-pasting the identical code is rather a bad practice.

Reply:
Addressed via [524e9e6](https://github.com/Haivision/srt/pull/2848/commits/524e9e6c0f0819b44929b8390e2a0ef38a6a3bdc).

It looks like FILE is the same, which is why I followed the existing pattern, but will leave that one.